test/test_datasets.py

```diff
@@ -2665,5 +2665,27 @@ def inject_fake_data(self, tmpdir: str, config):
         return num_images
 
 
+class RenderedSST2TestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.RenderedSST2
+    FEATURE_TYPES = (PIL.Image.Image, int)
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=("train", "valid", "test"))
+
+    def inject_fake_data(self, tmpdir: str, config):
+        root_folder = pathlib.Path(tmpdir) / "rendered-sst2"
+        image_folder = root_folder / config["split"]
+
+        num_images_per_class = 5
```

Comment:
To slightly increase robustness:

```suggestion
        num_images_per_class = {"train": 5, "test": 6, "val": 7}
```